libs/features/personalization/personalization.js

```diff
@@ -142,6 +144,13 @@ const COMMANDS = {
     el.insertAdjacentElement('beforebegin', createFrag(el, target, manifestId));
     el.classList.add(CLASS_EL_DELETE, CLASS_EL_REPLACE);
   },
+  replacemerchcard: (el, cardPath) => {
+    const paths = el.dataset[OVERRIDE_PATHS]?.split(',') || [];
+    if (!paths.includes(cardPath)) {
```

Comment:
`paths` could be undefined
```suggestion
    if (paths && !paths.includes(cardPath)) {
```